### PR TITLE
`exec` works in Python2 and Python3

### PR DESCRIPTION
**Pull request by annakovale**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,11 +3,8 @@
 
 _version_file = 'hooked/version.py'
 
-if sys.version_info.major < 3:
-    execfile(_version_file)
-else:
-    with open(_version_file) as vf:
-        exec(vf.read())
+with open(_version_file) as vf:
+    exec(vf.read())
```
Comment (marz619) [inline setup.py]: Not sure if we want to use `exec` here as they mean different things in python2 vs python3: see [this](https://stackoverflow.com/questions/15086040/behavior-of-exec-function-in-python-2-and-python-3/15087355#15087355)

Comment (marz619) [inline setup.py]: But then again, all we're trying to do is bring `__version__` into the local scope, so it might not matter.

Comment (annakovale) [inline setup.py]: @jonprindiville ^ what do you think?

Comment (jonprindiville) [inline setup.py]: Huh, so this works for us because we're _not_ trying to affect local variables?

Comment (marz619) [inline setup.py]: @jonprindiville the effect is different when you use `exec` within a function, then `local` vs `global` scope rules are different for py2 and py3 (at least that's my interpretation of that SO post)

Comment (jonprindiville) [inline setup.py]: Mm, then I think that this change is fine.

It's not within a function, and I doubt that our `setup.py` will get so complicated that we'll one-day need to wrap this in a function... YAGNI!

**Pull request merged**